src/makeInfinityAdder.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function adder(value) {
+    const sumResult = sum;
```

Comment:
lastValue or lastResult or lastSum would be clearer to me

Reply:
Agree

Reply:
fixed